packages/client/src/objectSet/ObjectSetListenerWebsocket.ts

```diff
@@ -175,16 +180,29 @@ export class ObjectSetListenerWebsocket {
       // will not be needed after our minimum version is 19 or greater.
       globalThis.crypto ??= (await import("node:crypto")).webcrypto as any;
     }
+
+    const objDef = await this.#client.ontologyProvider.getObjectDefinition(
+      objectType.apiName,
+    );
+
+    const objectProperties = properties.filter((p) =>
+      objDef.properties[p].type !== "geotimeSeriesReference"
+    );
+    const referenceProperties = properties.filter((p) =>
+      objDef.properties[p].type === "geotimeSeriesReference"
```

Comment:
I think my old comment is outdated now, but the question is the same. Where can I best ensure this is kept up to date? There shouldn't be any breaks if it isn't properly mapped(won't ever request it if not in this union and we can request these updates regularly)

Reply:
i dont know yet what we will do here but this is fine for now